jd4/daemon.py

```diff
@@ -130,16 +130,16 @@ def end(self, **kwargs):
 async def update_problem_data(session):
     ''' Invalidate all expired data. '''
     logger.info('Update problem data')
-    result = await session.judge_datalist(try_get_int('last_update_at'))
+    result = await session.judge_datalist(config.get('last_update_at', 0))
     for pid in result['pids']:
-      await cache_invalidate(pid['domain_id'], pid['pid'])
+      await cache_invalidate(pid['domain_id'], str(pid['pid']))
```

Comment:
`pid` is a dict, like `{'domain_id': 'system', 'pid': 1000}`.

The `pid['pid']` may be integer.